src/main/java/com/training/web/pages/flipkart/CartPage.java

```diff
@@ -0,0 +1,35 @@
+package com.training.web.pages.flipkart;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CartPage {
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    private WebDriver driver;
+    private WebDriverWait wait;
+
+    By totalPrice = By.xpath("//*[@id=\"container\"]/div/div[2]/div/div/div[2]/div[1]/div/div/div/div[4]/div/span/div/div");
+
+    public CartPage(WebDriver driver) {
+        this.driver = driver;
+        this.wait = new WebDriverWait(driver, 30);
+    }
+
+    public void name() {
+        String namep = driver.findElement(By.xpath("//*[@id=\"container\"]/div/div[2]/div/div/div[1]/div/div[2]/div/div[1]/div[1]/div[1]")).getText();
```

Comment:
flaky xpath - use a more optmised locator/xpath. Also this should not be hardcoded inside the function.